### DISCUSSION TITLE
File extraction help.

### DISCUSSION THREAD

**Discussion by nosideph** [lllyasviel/Fooocus#2790] [Q&A] [answered]
Hi there.

I'm hoping to get some insight on the issue. It's been 11 hours and the process is only at 28% completion. I have a 5600X, and I'm using WINRAR, and I just downloaded the file yesterday. 

![fooocus extract](https://github.com/lllyasviel/Fooocus/assets/70782707/d3166127-d081-489c-926e-9864c7e1a6ec)

Thanks a bunch!

**Comment by ILikeToasters** [comment, marked as answer]
Download and install 7zip: https://www.7-zip.org/download.html

**Reply by nosideph** [reply]
LOL. It took less than 5 minutes to extract the entire thing with 7Zip. Thanks so much!
